integration-tests/actions/vrf/vrfv2/contract_steps.go

```diff
@@ -207,12 +207,16 @@ func SetupVRFV2Contracts(
 		ReqsForTier5:                   big.NewInt(*vrfv2Config.ReqsForTier5)}
 
 	l.Info().Str("Coordinator", vrfContracts.CoordinatorV2.Address()).Msg("Setting Coordinator Config")
+	fallbackWeiPerUnitLink, ok := new(big.Int).SetString(*vrfv2Config.FallbackWeiPerUnitLink, 10)
```

Comment:
Would scientific notation work with this? like 1e16?

Reply:
now it should work